docs/js/main.js

```diff
@@ -93,5 +93,5 @@ $(function() {
 
   // Scroll now too in case we had opened the page on a hash, but wait a bit because some browsers
   // will try to do *their* initial scroll after running the onReady handler.
-  $(window).load(function() { setTimeout(function() { maybeScrollToHash(); }, 25); }); 
+  $(window).on('load', function() { setTimeout(function() { maybeScrollToHash(); }, 25); });
```

Comment:
This is actually the only change I seemed to need to make to the Spark javascript.

Reply:
Yep. This is the correct update based on the upgrade-guide.
- https://jquery.com/upgrade-guide/3.0/#breaking-change-load-unload-and-error-removed